packages/dd-trace/src/iitm.js

```diff
@@ -2,8 +2,21 @@
 
 const semver = require('semver')
 const logger = require('./log')
-
+const { moduleLoadStart } = require('./channel-itm')
```

Comment:
Each file calling DC directly would be better for decoupling.

Reply:
- [ ] Do not reuse channel-itm, each file with its channel